src/odemis/gui/util/test/img_test.py

```diff
@@ -1048,11 +1048,18 @@ def test_fake_canvas(self):
                         msg="Canvas are equal, which means there in no drawn ruler to be shown in the export")
 
 class TestSpatialExportPyramidal(unittest.TestCase):
-
-    def setUp(self):
-        self.app = wx.App()
-        data = numpy.zeros((2160, 2560), dtype=numpy.uint16)
-        dataRGB = numpy.zeros((2160, 2560, 4))
+    @classmethod
+    def setUpClass(cls):
+        # expected max number of pixels with a safety margin, derived from the MAX_RES_FACTOR in img.py
+        cls.expected_max_nmr_pixels = 23004160 * 0.95
+        cls.app = wx.App()
+
+    def _prepare_streams(self, fluo_stream_resolution, SEM_stream_resolution):
+        '''
```

Comment:
Nitpicky: Usually for docstrings you use triple double quotes (""")  instead of triple single quotes ('''). https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring